.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -25,11 +25,18 @@ body:
         - Queues
         - R2
         - Workers for Platforms
-        - Workers/Other
-        - Wrangler
+        - Constellation
+        - Wrangler core
+        - Other
   - type: input
     attributes:
-      label: What version of `Wrangler` are you using?
+      label: What version of Wrangler are you using?
```

Comment:
The Wrangler version is not necessarily appropriate for some issues now.
E.g. For C3 issues I would prefer to know the C3 version.
Can we change this?

Reply:
Perhaps we could say, please specify the versions of the tools you are using?

Reply:
👍